download.sh

```diff
@@ -0,0 +1,25 @@
+set -eux;
+
+ACCUMULO_VERSION="2.0.1"
```

Comment:
These versions are wrong for Accumulo 2.1.0-SNAPSHOT

Reply:
Yeah--looks like in the original Dockerfile, these versions were 2.1.0, 3.3.3, and 3.8.0